### ISSUE TITLE
Issue installing rtracklater with CURLINFO_FILETIME_T

### ISSUE THREAD

**Issue by thecatsmilk** [lawremi/rtracklayer#131]
I am currently attempting to install `rtracklayer` (R 4.4.1, Bioconductor 3.19). When installing, I am met with this error and can't seem to find a way to fix it:

```
In file included from /usr/include/x86_64-linux-gnu/curl/curl.h:2739:0,
                 from ucsc/net.c:7:
ucsc/net.c: In function ‘header_get_last_modified’:
ucsc/net.c:16:47: error: ‘CURLINFO_FILETIME_T’ undeclared (first use in this function); did you mean ‘CURLINFO_FILETIME’?
     CURLcode status = curl_easy_getinfo(curl, CURLINFO_FILETIME_T, &last_modified);
                                               ^
ucsc/net.c:16:47: note: each undeclared identifier is reported only once for each function it appears in
/usr/lib/R/etc/Makeconf:195: recipe for target 'ucsc/net.o' failed
make: *** [ucsc/net.o] Error 1
ERROR: compilation failed for package ‘rtracklayer’
* removing ‘/usr/lib/R/library/rtracklayer’

The downloaded source packages are in
	‘/tmp/RtmpyNzKvl/downloaded_packages’
Updating HTML index of packages in '.Library'
Making 'packages.html' ... done
Warning message:
In install.packages(...) :
  installation of package ‘rtracklayer’ had non-zero exit status
```

Any help would be greatly appreciated!

**Comment by sanchit-saini** [comment]
Hi @thecatsmilk 

It might be related to the issue [#123](https://github.com/lawremi/rtracklayer/issues/123) reported earlier, which was fixed with [#126](https://github.com/lawremi/rtracklayer/pull/126).
I tried to replicate it using the [Bioconductor Docker image](https://bioconductor.org/help/docker/) of the 3.19 version but could not replicate it.

This error happens if the system is using an older version (older than 7.59.0) of libcurl. You could try to install `rtracklayer` after updating libcurl.

**Comment by thecatsmilk** [comment]
Hi @sanchit-saini

Thanks for your reply! I'm not sure how I missed those threads you linked above; thanks for sharing them.

I upgraded to libcurl=8.3.0 and the issue has been solved. Thank you so much! You helped me avoid a massive headache.

Dan
